### ISSUE TITLE
Possibly an error in your mobilenet implementation 

### ISSUE THREAD

**Issue by TropComplique** [ildoonet/tf-pose-estimation#279]
Hi.
The original mobilenet v1 uses `relu6` activation:
https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.py#L472.

But you are not using activation function at all after depthwise layers:
https://github.com/ildoonet/tf-pose-estimation/blob/master/tf_pose/network_base.py#L180.

And you are using usual `relu` instead of `relu6` after pointwise layers:
https://github.com/ildoonet/tf-pose-estimation/blob/master/tf_pose/network_base.py#L193.

I believe that this could slightly decrease performance of the initial pretrained model.

**Comment by TropComplique** [comment]
Also, this is strange:
https://github.com/ildoonet/tf-pose-estimation/blob/master/tf_pose/network_mobilenet_thin.py#L49.

Combination `depthwise 1x1 conv layer` + `pointwise 1x1 conv layer` 
is equavalent to `conv 1x1 layer` (they must learn the same feature transformation).

**Comment by TropComplique** [comment]
Also, adding bias when using batch normalization is kinda useless:
https://github.com/ildoonet/tf-pose-estimation/blob/master/tf_pose/network_base.py#L196.

See this https://stackoverflow.com/a/46256860.

**Comment by shreyasrajesh** [comment]
Hi,
I noticed these issues too, did your performance significantly improve after making these changes?

**Comment by 1icas** [comment]
Purpose: train a model to get a good result for human pose estimation.  So, don't mind it's a mobilenet network structure or a "djwajdwa" network structure
